java-rx-smgrsdk/src/main/java/io/vantiq/svcconnector/ConnectorListener.java

```diff
@@ -33,21 +32,15 @@
 public class ConnectorListener {
 
     private final ObjectMapper mapper = new ObjectMapper();
-    VantiqStorageManager storageManager;
+    LocalMessageSender dispatcher;
```

Comment:
storage manager is now its own verticle listening on the bus. dispatching means dropping the request on the bus for it.